integration-cli/requirements.go

```diff
@@ -32,6 +33,13 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	Network = TestRequirement{
+		func() bool {
+			_, err := http.Get("http://google.com")
```

Comment:
I've mixed feeling about this. Can we `ifconfig` in --net=none container and figure this out from the interfaces? like ifconfig output contains "eth0" or something?

Also at least we should cache this result.